.github/workflows/mpi-operator-docker-image-publish.yml

```diff
@@ -26,6 +26,7 @@ jobs:
         with:
           images: |
             ${{ env.IMAGE_NAME }}
+            ghcr.io/${{ env.IMAGE_NAME }}
```

Comment:
@mahdikhashan You will need to store this image under `ghcr.io/kubeflow/mpi-operator/controller`.

Note that `IMAGE_NAME: mpioperator/mpi-operator` so using that variable will not work.

Reply:
done.